content/en/docs/concepts/observability-primer.md

```diff
@@ -12,11 +12,11 @@ description: >-
 ## What is Observability?
 
 Observability lets us understand a system from the outside, by letting us ask
-questions about that system without knowing its inner workings. Furthermore,
-allows us to easily troubleshoot and handle novel problems (i.e. "unknown
-unknowns”), and helps us answer the question, "Why is this happening?"
+questions about that system without knowing its inner workings. Furthermore, it
+allows us to easily troubleshoot and handle novel problems (i.e. unknown
+unknowns), and helps us answer the question, "Why is this happening?"
 
-In order to be able to ask those questions of a system, the application must be
+In order to be able to ask those questions about a system, the application must be
```

Comment:
Unrelated changes, not part of https://github.com/open-telemetry/opentelemetry.io/issues/1700